gas-service/src/lib.rs

```diff
@@ -53,6 +52,7 @@ pub trait GasService: events::Events {
     #[endpoint(payNativeGasForContractCall)]
     fn pay_native_gas_for_contract_call(
         &self,
+        sender: ManagedAddress,
```

Comment:
So any user can call this endpoint on behalf of any other user.

Reply:
Yes, to be consistent with the Solidity implementation:
https://github.com/axelarnetwork/axelar-cgp-solidity/blob/main/contracts/gas-service/AxelarGasService.sol

This allows a 3rd party to pay the gas on behalf of the user.